src/validation/amountValidation.js

```diff
@@ -0,0 +1,44 @@
+import AmountType from '../types/AmountType';
+/* eslint-disable import/prefer-default-export */
+// returns true if amount is invalid
+export function invalidAmountFields(amountType, min, max, setHelperText) {
```

Comment:
Instead of passing `setHelperText` I suggest returning the string itself so you can write tests against it.